services-js/commissions-app/src/server/email/EmailContent.ts

```diff
@@ -0,0 +1,66 @@
+import fs from 'fs';
+import Handlebars from 'handlebars';
+import mjml2html from 'mjml';
+
+require('./handlebars-helpers');
+
+export interface TemplateData {
+  commissions: Array<string>;
+  email: string;
+  newId: string;
```

Comment:
`applicationId`?

Reply:
That was my first impulse, but went back on it because `newId` is what the old code used... if you're thinking the same, then we‘ll happily use `applicationId` instead.

Reply:
I wouldn't spend any effort trying to match the naming of the old code. Assume that no one will look at it again.